src/helper/appJwtAuthentication.ts

```diff
@@ -66,9 +66,11 @@ export class AppJwtAuthentication {
 
       const client = new ClientCredentials(options);
 
-      const {access_token}: Token = await client.getToken({
-        scope: [],
-      });
+      const {access_token}: Token = (
+        await client.getToken({
+          scope: [],
+        })
+      ).token;
```

Comment:
Please seperate this out so the token property is not selected on a (possibly failing) promise
```suggestion
let access_token; 
try {
       access_token = await client.getToken({
          scope: [],
        })
] catch (error) {
// handle error
}
// get token etc.
```